datascience/tables.py

```diff
@@ -126,7 +138,16 @@ def read_table(cls, filepath_or_buffer, *args, **vargs):
         return cls.from_df(df)
 
     def _with_columns(self, columns):
-        """Create a table from a sequence of columns, copying column labels."""
+        """Create a table from a sequence of columns, copying column labels.
```

Comment:
As this function is using the Python convention for a private method, I don't think we should be adding any further documentation for this method. It would give the wrong impression to end users that this method can/should be used by them (and this method is simple enough for developers to understand on a quick skim)

Reply:
Done. Thank you for your response.